app/main.py

```diff
@@ -1,8 +1,36 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
After the opening bracket, write all the arguments on a new line; write the closing bracket on a new line too.